boards/wio_terminal/examples/clock.rs

```diff
@@ -1,5 +1,6 @@
 #![no_std]
 #![no_main]
+#![allow(static_mut_refs)]
```

Comment:
Is this something we'd want to look into replacing with `UnsafeCell`? Recent consensus is that static mut refs are considered Very Bad™ and essentially instant UB. Otherwise I'm happy with the PR, thanks!

Reply:
Or, maybe @Cooler1989 could take this on in their own PR.

Reply:
I've not dug in to the static mut thing, but FWIW have been using that in all my Rust firmware and not seen issues (of course, being careful to not violate the usual rules etc).  Maybe `UnsafeCell` is a good replacement in our examples, but in any case I feel we should handle that separately from the wio terminal update

Reply:
Fair enough! I think it's quite telling that it's now denied by default, but it can certainly be handled separately.